nixos/tests/home-assistant.nix

```diff
@@ -17,13 +17,16 @@ in {
             homeassistant = {
               name = "Home";
               time_zone = "UTC";
+              latitude = "0.0";
+              longitude = "0.0";
```

Comment:
null island! :)

Reply:
I had to put something because this is a required part of the config. If you happen to know Atlantis' coordinates, I'd be happy to take them instead :)